keyboards/converter/usb_usb/keymaps/via/keymap.c

```diff
@@ -0,0 +1,127 @@
+                      ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,
+    ______,           ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,               ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,
+    ______,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,  KC_F6,  KC_F7,  KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  ______,  ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  KC_UP,   ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,           ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  KC_LEFT, KC_DOWN, KC_RGHT, ______,  ______, ______, ______, ______,  ______,  ______,  ______,           ______,  ______,                                    ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,           ______,  ______,               ______,              ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,          ______,         ______,  ______,  ______,  ______,   MO(2),  ______,  ______,      ______,  ______,  ______,     ______,           ______,  ______,     ______,  ______
+    ),
+    [2] = LAYOUT_via(
+                      ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,
+     RESET,           ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,               ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,
+    ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,  ______,  ______,  ______,  ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______, EEP_RST,  ______,  ______,           ______,      ______,  ______,  ______,     ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,           ______,  ______,                                    ______,  ______,  ______,  ______,     ______,  ______,
+    ______,  ______,  ______,  ______,  ______,  ______, ______, ______, ______,  ______,  ______,  ______,           ______,  ______,               ______,              ______,  ______,  ______,  ______,     ______,  ______,
+     DEBUG,  ______,  ______,  ______,  ______,          ______,         ______,  ______,  ______,  ______,  ______,  ______,  ______,      ______,  ______,  ______,     ______,           ______,  ______,     ______,  ______
```

Comment:
```suggestion
    DB_TOGG,  ______,  ______,  ______,  ______,          ______,         ______,  ______,  ______,  ______,  ______,  ______,  ______,      ______,  ______,  ______,     ______,           ______,  ______,     ______,  ______
```

Reply:
Replaced DEBUG with QK_DEBUG_TOGGLE in the latest push.